Source/Tests/CAN Cyclic and Event Triggered Tests/targets.ini

```
@@ -0,0 +1,6 @@
+[Supported Targets]
+PharLap = 10.2.109.93
+Linux64PXI = vs-eco-slave-8880
```

The PXI target host name for the XNET ATS should be "VS-Core-8840QC".

```suggestion
PharLap = VS-Core-8840QC
Linux64PXI = VS-Core-8840QC
```